docs/cheaha/open_ondemand/index.md

```diff
@@ -25,3 +25,40 @@ For a more complete description of how to select resources, go [here](ood_layout
+
+This article tries to help you decide, by comparing both Jupyter Apps, looking at their strengths and weaknesses, and how they measure up to your needs, especially on Cheaha.
+
+### What They Have in Common
+
+Before listing out some of the differences, the following are similarities they have in common;
+
+1. They use the same underlying kernel system. (i.e. environments created for use with Jupyter Notebook can also be used in JupyterLab and vice-versa).
+1. They both support `.ipynb` notebooks.
+1. Allow interactive code execution, plotting, markdown, and data visualization.
+1. Can run Python, R, Julia, and other supported languages.
+1. they are both great for reproducing workflows.
+1. You can access both on Cheaha, selecting the configuration and resources required for your workflow.
+
+### Why Choose JupyterLab?
+
+JupyterLab is the next-generation interface for Jupyter. It's designed to be a full-featured web-based IDE. JupyterLab provides an interface for interactive and exploratory computing. Although it includes many features typical of traditional integrated development environments (IDEs), its primary focus remains on interactive and exploratory computing. The following are reasons why you may use JupyterLab over Jupyter Notebook.
+
+1. JupyterLab will be your preferred app, if you are conversant with or your workflow requires you to manage multiple notebooks, terminals, and other files. JupyterLab allows you to in a single tabbed interface, run your code, analyze logs, edit documentation, and move between tasks within your workflow more quickly.
+1. Projects that involve code, markdown, output files, and terminals simultaneously can be split into tabs within a browser tab, without switching browser tabs.
+1. If you want a more customizable and extensible environment, JupyterLab supports extensions out-of-the-box. If you need a Git GUI or a table of contents for navigating long notebooks? JupyterLab lets you add these features easily, making it more like a lightweight IDE than just a notebook viewer.
+
+![!Screenshot of JupyterLab interface with notebook + terminal + markdown viewer open side-by-side.](./images/jupyterlabsample.png)
+
+### Why Choose Jupyter Notebook?
+
+Jupyter Notebook offers the classic, streamlined experience that focuses on a single document at a time, the Jupyter Notebook interface is  ideal for teaching, learning, or lightweight research tasks. This interface, is what many users are already familiar with. The following are reasons why you may use Jupyter Notebook over JupyterLab.
```

Comment:
Split the first sentence into two at the first comma.

No comma needed after "This interface". It is helpful to say "The interface" because you are only talking about one. "This interface" implies there is more than one in this paragraph.

"Might use" instead of "may use". May implies you are giving permission. Might implies they will need to make choice.